chips/x86_q35/src/chip.rs

```diff
@@ -274,13 +283,22 @@ impl<I: InterruptService + 'static> Component for PcComponent<'static, I> {
 
         let syscall = Boundary::new();
 
+        // PS/2 inside the component
+        let ps2 =
+            unsafe { static_init!(crate::ps2::Ps2Controller, crate::ps2::Ps2Controller::new()) };
+        kernel::deferred_call::DeferredCallClient::register(ps2);
```

Comment:
Now is not that time, but at some point we need to stop adding new `static_init` here, which shouldn't have ever been here.